contracts/.env.example

```diff
@@ -3,6 +3,10 @@ RECIPIENT_REGISTRY_TYPE=simple
 
 # Supported values: simple, brightid
 USER_REGISTRY_TYPE=simple
+# clr.fund (prod) or CLRFundTest (testing)
+BRIGHTID_CONTEXT=clr.fund
+# BrightId node addr that signs verifications. Node One uses this one
+BRIGHTID_VERIFIER_ADDR=0xb1d71F62bEe34E9Fc349234C201090c33BCdF6DB
```

Comment:
More info can be found in the BrightID smart contract sample README, here: https://github.com/BrightID/BrightID-SmartContract

Shared doc with good technical info: https://docs.google.com/document/d/16-gqjfJLz7WSYVDHVQSFKK3Mj0GFWetxdgE9nQuMS9o